addons/im_livechat/models/discuss_channel.py

```diff
@@ -7,6 +7,15 @@
 from markupsafe import Markup
 
 
+def livechat_channel_predicate(channel):
```

Comment:
Maybe the name should be more in regards to what is does: `only_livechat_channel` or something like that?

Reply:
what about `is_livechat_channel`?

```py
Store.Attr("livechat_operator_id", predicate=is_livechat_channel)
```